android/java/org/chromium/chrome/browser/crypto_wallet/fragments/SwapBottomSheetDialogFragment.java

```diff
@@ -100,7 +100,7 @@ public void onClick(View view) {
 
         if (!JavaUtils.anyNull(mWalletModel, mNetworkInfo)) {
             BuySendSwapActivity.ActivityType finalActivityType = activityType;
-            mWalletModel.getNetworkModel().setNetwork(
+            mWalletModel.getNetworkModel().setNetworkForSelectedAccountOnActiveOrigin(
```

Comment:
Can we have network picker for Buy and Send to be independent of backend's network for account? Just a network for new transaction.